langchain-core/src/tracers/base.ts

```diff
@@ -392,7 +392,11 @@ export abstract class BaseTracer extends BaseCallbackHandler {
     return run;
   }
 
-  async handleToolEnd(output: string, runId: string): Promise<Run> {
+  async handleToolEnd(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    output: string | Record<string, any>,
```

Comment:
Thought we weren't doing this yet?

Reply:
whoops, meant to remove when I merged branches. thanks